### PR TITLE
Core/Unit: Update Model Form

### LINKED ISSUE #18300: [Master] Druid forms model issues
[//]: # (***************************************************************)
[//]: # (** DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED **)
[//]: # (***************************************************************)

**Description:** 
Morphing into certain druid forms with certain hair color combinations (and whatnot) results in texture-blended models.

**Current behaviour:** (Tell us what happens.) 
Player gets turned into some sort of texture-blended bear-like model.

**Expected behaviour:** (Tell us what should happen instead.) 
Player should be turned into Bear form like in retail.

**Steps to reproduce the problem:**

1. Create Night Elf Male Druid with Purple hair (last option)
2. Turn into bear with Bear Form
3. See yourself getting turned into a texture-blended bear model

**Branch(es):** 3.3.5 / master (Tell us which branch(es) this issue affects.) master

**TC rev. hash/commit:**  https://github.com/TrinityCore/TrinityCore/commit/cb41b67a7b0d5e4436210d3d54e3b1f37724d239

**TDB version:**  703.00

**Operating system:**  Windows 10 64-bit

[//]: # (This template is for problem reports. For other types of report, edit it accordingly.)
[//]: # (If this is a crash report, include the crashlog with https://gist.github.com/)
[//]: # (For fixes containing C++, create a Pull Request.)

### PR DESCRIPTION
**Pull request by ghost**
Update Model Form for Druid

**Changes proposed:**

-  Fix default bear model for Night Elf 
-  Add Form for Travel Form (Stag)
-  Add Form for (Glyph of the Sentinel, Glyph of Stars, Glyph of the Orca, Glyph of the Cheetah & Glyph of the Doe)

**Target branch(es):** master

**Issues addressed:** Closes #18300

**Tests performed:** Builded and tested in game

### CODE REVIEW

**Comment by jackpoz** [comment]
Please leave codestyle changes out of PRs that change logic

File: src/server/game/Entities/Unit/Unit.cpp [hunk 1]
```diff
@@ -14618,8 +14641,33 @@ uint32 Unit::GetModelForForm(ShapeshiftForm form) const
                         break;
                 }
                 break;
+            }
+            case FORM_AQUATIC_FORM:
+                if (HasAura(114333)) // Glyph of the Orca
+                    return 4591;
```
Comment (Shauren) [inline src/server/game/Entities/Unit/Unit.cpp]: I'm sure no code from TRAVEL_FORM case should be executed if unit does not have this aura

Comment (DoctorKraft) [inline src/server/game/Entities/Unit/Unit.cpp]: Oops ! I have missed that. It's correct now, i have try

**Comment by DoctorKraft** [comment]
Added Glyph of the Ursol Chameleon & Glyph of the Feral Chameleon, I'm not sure for method but it's working. If you know better solution, let me know.

I have also delete Glyph of Stars (Moonkin Form) because he break animation, i have a other method for implented in new PR.

**Pull request merged**